packages/cacti-plugin-copm-corda/Makefile

```diff
@@ -0,0 +1,39 @@
+WORKSPACE ?= $(shell realpath `pwd`/../../)
+
+build-local-deps:
+	cd $(WORKSPACE)/packages/cacti-copm-core/src/main/protos-kt && \
+	./gradlew build && \
+	cd $(WORKSPACE)/packages/cacti-copm-core && \
+	make corda-setup
+	mkdir -p localdeps && \
+	cp $(WORKSPACE)/packages/cacti-copm-core/src/main/protos-kt/build/libs/copm-corda-*jar localdeps 
+	cp $(WORKSPACE)/weaver/common/protos-java-kt/build/libs/protos-java-kt-*.jar localdeps
+	cp $(WORKSPACE)/weaver/sdks/corda/build/libs/weaver-sdk-corda-*.jar localdeps
+	cp $(WORKSPACE)/weaver/core/network/corda-interop-app/interop-workflows/build/libs/interop-workflows-*.jar localdeps
+	cp $(WORKSPACE)/weaver/core/network/corda-interop-app/interop-contracts/build/libs/interop-contracts-*.jar localdeps
+	cp $(WORKSPACE)/weaver/samples/corda/corda-simple-application/contracts-kotlin/build/libs/contracts-kotlin-*.jar localdeps
+	cp $(WORKSPACE)/weaver/samples/corda/corda-simple-application/workflows-kotlin/build/libs/workflows-kotlin-*.jar localdeps
+
+server-docker-image:
+	DOCKER_BUILDKIT=1 docker build . --tag copm-corda 
+
+setup: build-local-deps server-docker-image
+
+
+pledge-network:  
```

Comment:
Would suggest moving these makefile targets, like starting network and other components to tests, and those should be ideally in examples directory

Reply:
Makefiles have been moved to cacti-copm-test and the cacti-plugin-copm-corda/src/test/kotlin directories.